src/prototype_room_init.js

```diff
@@ -465,14 +465,19 @@ Room.prototype.setup = function() {
 
   this.costMatrixPathCrossings(exits);
 
+  const minerals = this.find(FIND_MINERALS);
```

Comment:
it will conflict with my mineral PR

Reply:
I know. I've already resolved the merge conflicts on my side. I'll push a version with no conflicts as soon as I get that nailed down.